lib/abci/handlers/beginBlockHandlerFactory.js

```diff
@@ -20,6 +21,7 @@ function beginBlockHandlerFactory(
   blockchainState,
   blockExecutionDBTransactions,
   blockExecutionState,
+  appVersion,
```

Comment:
let's call it protocolVersion? Tendermint's name is kinda confusing. No idea why they called it like this.

Reply:
Yes, I tried to do that as well, if you look at the commit history. 
But it was way to much confusion that I decided to revert back, because at some point we need to match it with Tendermint name :/ 
I can do it if you really want that though.

Reply:
I wouldn't touch such things in our fork in order to be as much compatible as possible. The main goal of our fork atm is to add additional dash-specific logic but not optimize/improve/break compatibility with upstream.